.cirrus.yml

```diff
@@ -109,12 +109,12 @@ task:
   install_script:
     - apt-get update
     - apt-get -y install curl jq bc
-    - python3 -m pip install --user --upgrade pip
+    - python3 -m pip install --user --upgrade pip --break-system-packages
```

Comment:
Would be cleaner IMO to set the env var instead, to avoid the duplications: `PIP_BREAK_SYSTEM_PACKAGES=1`

https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-break-system-packages